libraries/app/api.cpp

```diff
@@ -352,7 +352,8 @@ namespace graphene { namespace app {
     {
        FC_ASSERT( _app.chain_database() );
        const auto& db = *_app.chain_database();
-       FC_ASSERT( limit <= 100 );
+        uint32_t max_account_history_operations_limit=_app.get_options().max_account_history_operations_limit;
```

Comment:
I think you can add `const` here. Also please use codebase indentation of 3 spaces.